tests/cli/manifests/test_python_version_resolution.py

```diff
@@ -0,0 +1,275 @@
+import pytest
+from unittest import mock
+
+from soar_sdk.cli.manifests.processors import ManifestProcessor
+
+
+class TestGetTargetPythonVersions:
+    """Test the get_target_python_versions method that determines which Python versions to build for."""
+
+    @pytest.fixture
+    def mock_pyproject_toml(self, tmp_path):
+        """Create a temporary directory for test pyproject.toml files."""
+        return tmp_path
+
+    def test_both_versions_when_no_constraint(self, mock_pyproject_toml):
+        """When no requires-python is specified, should return all SDK-supported versions."""
+        pyproject_content = """
+[project]
+name = "test-app"
+version = "1.0.0"
+"""
```

Comment:
`textwap.dedent` helps clean these up:


```suggestion
        pyproject_content = textwrap.dedent(
            """
            [project]
            name = "test-app"
            version = "1.0.0"
            """
        )
```